ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionTablettEllerInsulinMessage.java

```diff
@@ -58,4 +60,10 @@ public static CertificateDataElement toCertificate(int index, CertificateTextPro
             )
             .build();
     }
+
+    private static Boolean setVisibility(Diabetes diabetes) {
```

Comment:
Consider naming it getVisibility(..) instead of set, as the code doesn't set anything.